openpype/pipeline/colorspace.py

```diff
@@ -455,14 +456,15 @@ def _get_imageio_settings(project_settings, host_name):
 
     Args:
         project_settings (dict): project settings.
-                                           Defaults to None.
+                                 Defaults to None.
         host_name (str): host name
 
     Returns:
         tuple[dict, dict]: image io settings for global and host
     """
     # get image io from global and host_name
     imageio_global = project_settings["global"]["imageio"]
-    imageio_host = project_settings[host_name]["imageio"]
+    # host is optional, some might not have any settings
+    imageio_host = project_settings[host_name].get("imageio", {})
```

Comment:
```suggestion
    imageio_host = project_settings.get(host_name, {}).get("imageio", {})
```

Reply:
Yes please! Technically there's no **hard** requirement that the host has project settings at all.

Reply:
I was thinking this might happen ;) Thanks!